task_5/Toma.Lobanova/src/main/java/ua/od/hillel/SequenceTask3/Sequence.java

```diff
@@ -0,0 +1,10 @@
+package ua.od.hillel.SequenceTask3;
+
+import java.util.stream.Stream;
+import java.util.stream.Collectors;
+
+public class Sequence {
+    public static void main(String... args){
+        System.out.println(Stream.iterate(90,item -> item-5).limit(90/5).filter(item -> item>=0).collect(Collectors.toList()));
```

Comment:
Use forEach operation instead of collect. You don't need System.out.println in the beginning then